modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaGetCategoryIdMethodCheck.testjava

```diff
@@ -0,0 +1,11 @@
+package com.liferay.source.formatter.dependencies.upgrade;
+
+public class UpgradeJavaGetCategoryIdMethodCheck {
+
+	public void method(AssetCategory<?> assetCategory) {
+		String assetCategoryClassName = assetCategory.getCategoryId;
+
+		String assetCategoryClassName = AssetCategory::getCategoryId;
```

Comment:
Please change to

```
		UnsafeFunction<AssetCategory, Long, Throwable> getCategoryId = AssetCategory::getCategoryId;
```

This statement isn't calling the method, is returning the method at self.

Reply:
Resolved